frontend/src/entry/model/settings/ModelAccess.tsx

```diff
@@ -30,6 +31,7 @@ export default function ModelAccess({ model }: ModelAccessProps) {
   const [userListQuery, setUserListQuery] = useState('')
   const [loading, setLoading] = useState(false)
   const [errorMessage, setErrorMessage] = useState('')
+  const [roleError, setRoleError] = useState('')
```

Comment:
This isn't needed because we already have `errorMessage` that we can use to display your new errors being returned. We always use a `MessageAlert` to display errors to the user, so opting for a `Tooltip` here adds inconsistency to our app.

I suggest using `errorMessage` to handle the displaying of the error and have a boolean `isEntityError` to handle disabling the button.